resources/home/dnanexus/generate_workbook/utils/utils.py

```
@@ -234,7 +234,7 @@ def gnomad(self, value, build) -> str:
         if build == 37:
             url = f"{self.urls['gnomad_base_url']}?dataset=gnomad_r2_1"
         elif build == 38:
-            url = f"{self.urls['gnomad_base_url']}?dataset=gnomad_r3"
+            url = f"{self.urls['gnomad_base_url']}?dataset=gnomad_r4"
         else:
             return value[column]
```

Ruff (0.12.2)

Undefined name `column`

(F821)